google/cloud/forseti/services/model/importer/importer.py

```diff
@@ -369,7 +369,10 @@ def _store_iam_policy(self, policy):
                 msg = 'Role reference in iam policy not found: {}'.format(role)
                 self.model.add_warning(msg)
                 continue
-            for member in binding['members']:
+
+            #binding['members'] can have duplicate ids
```

Comment:
Please add a space after the pound sign.  Can you also add more detail to the comment, about why the member must be unique for building the model?  i.e. why we need to make this unique?